.github/workflows/helm-check.yaml

```diff
@@ -37,18 +37,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.0.0-rc.2
-        with:
-          command: lint
-          config: install/kubernetes/chart-testing.yaml
+      - name: Run helm int
+        run: |
+          cd install/kubernetes/cilium
+          helm lint --with-subcharts --values values.yaml
```

Comment:
Can we assume `helm` is always available if we no longer use `helm/chart-testing-action`?

Reply:
yup, I have checked the software in ubuntu-latest. Helm 3 is already shipped with github action.

https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md